### DISCUSSION TITLE
Port Forwarding

### DISCUSSION THREAD

**Discussion by PCRover** [wavelog/wavelog#1085] [Q&A] [answered]
What ports do I need to forward through my firewall to get to Wavelog? Just port 80 or is there something additional? 
I tried forwarding port 8080 outside to port 80 inside and it is coming up blank. I have several other port forwards to other applications that work just fine. Only Wavelog is giving me issues using apache2.

**Comment by HB9HIL** [comment, marked as answer]
It depends on your setup. Are you using docker or did you install wavelog with git?

**Reply by PCRover** [reply]
It is a git install on a proxmox vm on debian. 

I also did a docker install on a Synology NAS which would give me a login page remotely but would not accept my login. It kept going back to the login page.

**Reply by HB9HIL** [reply]
You should never forget. We only provide the software. How you set up your webserver, securing ports and firewall is beyond our power. 

There are plenty of instructions, tutorials and youtube videos how to selfhost services. 

When you installed Wavelog with git, I **can't** tell you which port you need to forward in your router. It depends on your webserver configuration. I assume it's port 80 and/or 443.

**Reply by HB9HIL** [reply]
Wavelog is no "setup.exe" software. You have to provide some knowledge about Linux and Webservers. 

Check our Wiki about default[ webserver configurations](https://github.com/wavelog/wavelog/wiki/Webserver-Configurations). This may help you to at least reach wavelog in your LAN

**Reply by PCRover** [reply]
Just wanted to know if any special ports were in use that needed to be forwarded through a firewall that I missed somehow. Not asking for webserver setup help. Clearly the answer is no judging by your response. Thanks for your time.

**Reply by HB9HIL** [reply]
Sorry then I did not get that right. 

There are no special ports. Use 80 or 443 pointing to the installation path
